Specs/DateTime/Hindi/DateTimeExtractor.json

```diff
@@ -562,5 +562,246 @@
         "Length": 21
       }
     ]
+  },
+  {
+    "Input": "मैं कल सुबह 9 बजे वापस जाऊंगा",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "कल सुबह 9 बजे",
+        "Type": "datetime",
+        "Start": 4,
+        "Length": 13
+      }
+    ]
+  },
+  {
+    "Input": "मैं कल सुबह 9 बजे वापस जाऊँगा",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "कल सुबह 9 बजे",
+        "Type": "datetime",
+        "Start": 4,
+        "Length": 13
+      }
+    ]
+  },
+  {
+    "Input": "मैं कल 9 बजे वापस जाऊंगा",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "कल 9 बजे",
+        "Type": "datetime",
+        "Start": 4,
+        "Length": 8
+      }
+    ]
+  },
+  {
+    "Input": "इस शुक्रवार को दोपहर एक बजे",
+    "NotSupportedByDesign": "javascript,python,java",
+    "Results": [
+      {
+        "Text": "इस शुक्रवार को दोपहर एक बजे",
+        "Type": "datetime",
+        "Start": 0,
+        "Length": 27
+      }
+    ]
+  },
+  {
+    "Input": "दोपहर 12:30 बजे शुक्रवार के लिए लंच जोड़ें",
+    "Comment": "Disable this for now because of new features in .NET",
```

Comment:
This is not disabled. Does this comment apply? And which feature was it?

Reply:
this comes from the English spec (where the test case is not disabled as well). 
I will remove both.